File: backend/components/metiers/administration_publique/guides/fallback/samples/sample_fallback_case.test.js
// sample_fallback_case.test.js – Tests unitaires et edge cases JS
const { sampleFallbackCase } = require('./sample_fallback_case');

test('sampleFallbackCase structure', () => {
  expect(typeof sampleFallbackCase).toBe('object');
  expect(sampleFallbackCase).toHaveProperty('id');
  expect(sampleFallbackCase).toHaveProperty('description');
  expect(sampleFallbackCase).toHaveProperty('status');
});

test('sampleFallbackCase status', () => {
  expect(['compliant', 'partial', 'non-compliant']).toContain(sampleFallbackCase.status);
});

File: backend/components/metiers/administration_publique/guides/fallback/samples/sample_fallback_case.test.py
# sample_fallback_case.test.py – Tests unitaires et edge cases Python
from .sample_fallback_case import sample_fallback_case

def test_sample_fallback_case_structure():
    assert isinstance(sample_fallback_case, dict)
    assert 'id' in sample_fallback_case
    assert 'description' in sample_fallback_case
    assert 'status' in sample_fallback_case

def test_sample_fallback_case_status():
    assert sample_fallback_case['status'] in ['compliant', 'partial', 'non-compliant']
